openedx/core/djangoapps/programs/utils.py

```diff
@@ -142,6 +170,11 @@ def progress(self, programs=None, count_only=True):
             list of dict, each containing information about a user's progress
                 towards completing a program.
         """
+        now = datetime.datetime.now(utc)
+        enrolled_run_modes = {}
+        for enrollment in self.enrollments:
+            enrolled_run_modes[unicode(enrollment.course_id)] = enrollment.mode
```

Comment:
i could but the rest of the class doesn't need it.  
i feel like class level variables make sense when they are used by multiple methods?